src/App.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import './App.scss';
+
+import Clock from './components/Clock';
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 0,
+  };
+
+  showClock = () => {
+    this.setState({ isClockVisible: true });
+  };
+
+  hideClock = () => {
+    this.setState({ isClockVisible: false });
+  };
+
+  randomName = () => {
+    this.setState({ clockName: Math.random() });
+  };
+
+  render() {
+    const { isClockVisible, clockName } = this.state;
+
+    return (
+      <div className="App">
```

Comment:
```suggestion
      <div className="Clock">
```

These styles used not for the whole App, but just for the 'Clock' component. Not critical for this task but pay attention to it in future